e2e-tests/production-runtime/cypress/integration/resource-loading-resilience.js

```diff
@@ -63,52 +63,46 @@ const runTests = (testNameSuffix = `Unknown scenario`) => {
   })
 }
 
-const getTestNameSuffix = (scenario, args) => {
-  if (scenario === `blockAssetsForChunk` && args.chunk === `app`) {
-    return `Blocked "app" chunk`
-  } else if (scenario === `blockAssetsForPage`) {
-    return `Blocked "${args.filter}" for "${args.pagePath}"`
-  }
-
-  return undefined
-}
-
-const runBlockedScenario = (scenario, args) => {
-  describe(`Block resources`, () => {
-    before(done => {
-      cy.task(`restoreAllBlockedResources`).then(() => {
-        cy.task(scenario, args).then(() => {
-          done()
+const runBlockedScenario = args => {
+  before(() => {
+    cy.task("getAssetsForPage", {
+      pagePath: args.pagePath,
+      filter: args.filter,
+    }).then(urls => {
+      for (const url of urls) {
+        cy.intercept(url, {
+          statusCode: 404,
+          body: "",
         })
```

Comment:
does those stay blocked for following tests? Do we need to "unintercept" those so next steps don't "inherit" setup with resources blocked from previous runs?